### PR TITLE
docs(README): clarify usage

### PR DESCRIPTION
**Pull request by IstoraMandiri**
To help avoid confusion about usage re: #209

### CODE REVIEW

**Comment by MattIPv4** [comment]
@gr2m Thoughts here? I don't particularly see the need to state requiring a new repo here, the action would work in any repo assuming the tweets directory is present (e.g. a general repo dedicated to everything social-related for a project).

**Comment by IstoraMandiri** [comment]
Fair comment, I've updated the wording. How is this?

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ The Twitter Ads API we currently use is the `v8` version.
 
 ## Setup
 
+Unless you wish to contribute to this project, you don't need to fork this repository. Instead, complete the following steps within your own new or existing GitHub respositry.
```
Comment (MattIPv4) [inline README.md:43]:
```suggestion
Unless you wish to contribute to this project, you don't need to fork this repository. Instead, you can make use of this GitHub Action from the comfort of your own repository (either a new one, or one you already have) by creating a GitHub Actions workflow following these steps:
```

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Some copy tweaks

File: README.md (hunk 1 above)
Comment (IstoraMandiri) [reply README.md:43]: Very good!

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Review by gr2m — APPROVED** [review]
Thanks a lot!

**Pull request merged**